internal/cmprefimpl/cmppg/pg.go

```diff
@@ -46,6 +48,23 @@ const (
 
 // Unary runs a batch of unary operations on a geometry.
 func (p BatchPostGIS) Unary(g geom.Geometry) (UnaryResult, error) {
+	var i int
```

Comment:
I noticed a little bit of intermittent instability with the new postgis image -- adding some retry seems to fix the issue.

Reply:
> I noticed a little bit of intermittent instability with the new postgis image -- adding some retry seems to fix the issue.

What do you think about adding ☝🏼 somewhere here? As a reader, I would be wondering what was the motivation for adding this retry logic (and could be hard to find by searching through PRs).

Reply:
Good call, added in b45949abf5f21c5a0284b741d8b4db651a7256f2.